shell/browser/extensions/electron_extension_loader.cc

```diff
@@ -110,6 +113,26 @@ void ElectronExtensionLoader::FinishExtensionLoad(
   if (extension) {
     extension_registrar_.AddExtension(extension);
   }
+
+  // Write extension install time to ExtensionPrefs. This is required by
+  // WebRequestAPI which calls extensions::ExtensionPrefs::GetInstallTime.
+  //
+  // Implementation for writing the pref was based on
+  // PreferenceAPIBase::SetExtensionControlledPref.
+  {
+    ExtensionPrefs* extension_prefs = ExtensionPrefs::Get(browser_context_);
+    ExtensionPrefs::ScopedDictionaryUpdate update(
+        extension_prefs, extension.get()->id(),
+        extensions::pref_names::kPrefPreferences);
+    auto preference = update.Create();
+    const base::Time install_time = base::Time().Now();
+    std::unique_ptr<base::Value> install_time_str =
+        base::Value(base::NumberToString(install_time.ToInternalValue()))
+            .CreateDeepCopy();
+    preference->SetWithoutPathExpansion("install_time",
+                                        std::move(install_time_str));
```

Comment:
`SetWithoutPathExpansion` is deprecated, see https://source.chromium.org/chromium/chromium/src/+/master:base/values.h;l=661;drc=2a06d6e94a216850e057c4ba7d36c64032448c77

```suggestion
    preference->SetStringKey("install_time", base::NumberToString(install_time.ToInternalValue()));
```